client/payment-methods/index.js

```diff
@@ -124,6 +112,56 @@ const UpeDisableError = () => {
 	);
 };
 
+const UpeSetupBanner = () => {
+	const [ , setIsUpeEnabled ] = useIsUpeEnabled();
+
+	const handleEnableUpeClick = () => {
+		setIsUpeEnabled( true ).then( () => {
+			window.location.href = addQueryArgs( 'admin.php', {
+				page: 'wc-admin',
+				task: 'woocommerce-payments--additional-payment-methods',
+			} );
+		} );
+	};
+
+	return (
+		<>
+			<CardDivider />
+			<CardBody className="payment-methods__express-checkouts">
```

Comment:
This now happens. I think it's due to the way the components are conditionally displayed on the settings page.

![Markup on 2021-07-16 at 07:53:55](https://user-images.githubusercontent.com/1759681/125904376-730b98e9-08e1-4e7e-a419-44533804adfd.png)

Reply:
I think this issue is being address in these 2 places separately: 
https://github.com/Automattic/woocommerce-payments/pull/2423
https://github.com/Automattic/woocommerce-payments/pull/2456

Reply:
That's correct - the section should display the "card" payment method when UPE is disabled. Those 2 PRs will introduce the correct behavior.

Reply:
Oh, I see! Thanks for pointing it out!